README.md

```diff
@@ -37,6 +37,13 @@ pip install transformers==4.19.2 diffusers invisible-watermark
 pip install -e .
 ``` 
 
+Follow this steps to get correctly the file named sd-v1-4.ckpt
+1. Go to this link https://huggingface.co/
+2. Signup and login
+3. Then go to this page https://huggingface.co/CompVis/stable-diffusion-v-1-4-original
+4. Click that you agree the terms (if you agree the terms, else you can't download it)
```

Comment:
```suggestion
4. Click that you agree the terms (if you agree the terms, else you can't download it) and hit the "Agree and access repository" button
```
I just followed this word by word since I was stuck at this stage of the install and I did follow it literally so didn't hit the button, I guess there are guys like me out there so I'd suggest to add this precision.